### PR TITLE
Add cloud logger to installation script

### LINKED ISSUE #19575: Add new `wazuh/shared/` folder of wodles to wazuh installation function
| Affected integration |
|---|
| AWS, GCloud, Azure|

## Description

As part of #16717, with the integration of the new logger the `inst-function.sh` needs to be modified to include the `wodles/shared/` folder into wazuh installation function. 

## Tasks
- [x] Test in a manager.
- [x] Test in an agent.
- [x] Unit tests without failures. Updated if there are any relevant changes.
- [x] Integration tests without failures. Updated if there are any relevant changes.
- [x] Update the documentation if necessary.
- [x] Add entry to the changelog if necessary.

### PR DESCRIPTION
**Pull request by EduLeon12**
|Related issue|
|---|
|Closes #19575 |

## Description

As explained in #19575, Thie Pr tends to modify the installation script to include the cloud logger.

### CODE REVIEW

File: src/init/inst-functions.sh [hunk 1]
```diff
@@ -1141,6 +1141,11 @@ InstallServer()
     ${INSTALL} -m 0750 -o root -g ${WAZUH_GROUP} ../wodles/__init__.py ${INSTALLDIR}/wodles/__init__.py
     ${INSTALL} -m 0750 -o root -g ${WAZUH_GROUP} ../wodles/utils.py ${INSTALLDIR}/wodles/utils.py
 
+        # Install the Wazuh cloud logger
+    ${INSTALL} -d -m 0750 -o root -g ${WAZUH_GROUP} ${INSTALLDIR}/wodles/shared
```
Comment (GGP1) [inline src/init/inst-functions.sh:1145]:
```suggestion
    # Install the Wazuh cloud logger
    ${INSTALL} -d -m 0750 -o root -g ${WAZUH_GROUP} ${INSTALLDIR}/wodles/shared
```

File: src/init/inst-functions.sh [hunk 2]
```diff
@@ -1222,6 +1227,11 @@ InstallAgent()
...
```
Comment (GGP1) [inline src/init/inst-functions.sh:1231]:
```suggestion
    # Install the Wazuh cloud logger
    ${INSTALL} -d -m 0750 -o root -g ${WAZUH_GROUP} ${INSTALLDIR}/wodles/shared
```

**Review by GGP1 — CHANGES_REQUESTED** [review]
The installation was successful and the cloud logger was found at `/var/ossec/wodles/shared/wazuh_cloud_logger.py`. Just suggested to align comments with commands.

**Review by GGP1 — APPROVED** [review]
LGTM

**Review by Selutario — APPROVED** [review]

**Pull request merged**